inc/class-addon-manager.php

```diff
@@ -266,11 +272,101 @@ protected function convert_subscription_to_plugin( $subscription ) {
 			'download_link' => $subscription->product->download,
 			'package'       => $subscription->product->download,
 			'sections'      => [
-				'changelog' => $subscription->product->changelog,
+				'changelog' => $changelog,
+				'support'   => $this->get_support_section(),
 			],
+			'icons'         => [
+				'2x' => $this->get_icon( $subscription->product->slug ),
+			],
+			'banners'       => $this->get_banners( $subscription->product->slug ),
+			'tested'        => isset( $yoast_seo_data->update->tested ) ? $yoast_seo_data->update->tested : '',
+			'requires_php'  => isset( $yoast_seo_data->update->requires_php ) ? $yoast_seo_data->update->requires_php : '',
 		];
 	}
 
+	/**
+	 * Get the Yoast SEO plugin data.
+	 *
+	 * @return object Yoast SEO plugin data.
+	 */
+	protected function get_yoast_seo_data() {
+		static $plugin_updates, $yoast_seo_data;
+		if ( ! isset( $plugin_updates ) ) {
+			$plugin_updates = get_plugin_updates();
+		}
+
+		if ( ! isset( $yoast_seo_data ) ) {
+			foreach ( $plugin_updates as $file => $data ) {
+				$free_file        = 'wp-seo.php';
+				$free_file_length = strlen( $free_file );
+				if ( substr( $file, - $free_file_length ) === $free_file ) {
+					$yoast_seo_data = (object) _get_plugin_data_markup_translate( $file, (array) $plugin_updates[ $file ], false, true );
+				}
```

Comment:
Regarding:
> When Yoast SEO free is installed, compatibility data will show, if it's not installed, this will not show.

Not sure this is the intended behavior but seems to me this will work only if Yoast SEO itself has available updates. Otherwise, `$yoast_seo_data` will be `null` and the compatibility data of the add-ons won't be available:

<img width="977" alt="Screenshot 2020-08-25 at 11 12 31" src="https://user-images.githubusercontent.com/1682452/91158414-e3eb8e80-e6c6-11ea-803d-f01e38604e9a.png">

@jdevalk is this the intended behavior?